packages/evergarden/src/Button/examples.js

```diff
@@ -1,97 +1,48 @@
-import { Button, ButtonGroup } from 'evergarden'
+import Vue from 'vue'
+import { Button } from 'evergarden'
+import { addDecorator } from '@storybook/vue'
 import { action } from '@storybook/addon-actions'
+import { withKnobs, text, select, boolean } from '@storybook/addon-knobs'
 
-export default { title: 'Button' }
-
-export const colors = () => ({
-  render() {
-    return (
-      <ButtonGroup>
-        <Button variantColor="green">Button</Button>
-
-        <Button variantColor="red">Button</Button>
-
-        <Button variantColor="cyan">Button</Button>
-
-        <Button variantColor="orange">Button</Button>
-
-        <Button
-          variantColor="blue"
-          color="gray.300"
-          borderRadius="lg"
-          _hover={{ color: 'gray.700' }}
-        >
-          Compose
-        </Button>
-      </ButtonGroup>
-    )
-  }
-})
-
-export const variants = () => ({
-  render(h) {
-    return (
-      <ButtonGroup>
-        <Button variantColor="green" variant="solid">
-          Button
-        </Button>
+addDecorator(withKnobs)
 
-        <Button variantColor="green" variant="outline">
-          Button
-        </Button>
+const variants = {
+  label: 'Variants',
+  options: ['green', 'red', 'cyan', 'orange', 'blue'],
+  defaultValue: 'green'
+}
 
-        <Button variantColor="green" variant="ghost">
-          Button
-        </Button>
+const sizes = {
+  label: 'Sizes',
+  options: ['xs', 'sm', 'md', 'lg'],
+  defaultValue: 'md'
+}
 
-        <Button variantColor="green" variant="link">
-          Button
-        </Button>
-      </ButtonGroup>
-    )
-  }
-})
-
-export const sizes = () => ({
-  render(h) {
-    return (
-      <ButtonGroup>
-        <Button variantColor="green" size="xs">
-          Button
-        </Button>
-
-        <Button variantColor="green" size="sm">
-          Button
-        </Button>
-
-        <Button variantColor="green" size="md">
-          Button
-        </Button>
-
-        <Button variantColor="green" size="lg">
-          Button
-        </Button>
-      </ButtonGroup>
-    )
-  }
-})
+export default { title: 'Button' }
 
-export const events = () => ({
-  render(h) {
-    return (
-      <ButtonGroup>
-        <Button onClick={action('hello')}>Click Me!</Button>
-      </ButtonGroup>
-    )
-  }
+export const button = () => ({
+  components: { Button },
+  props: {
+    variant: {
+      default: select(variants.label, variants.options, variants.defaultValue)
+    },
+    size: {
+      default: select(sizes.label, sizes.options, sizes.defaultValue)
+    },
+    text: {
+      default: text('Button text', 'Click me!')
+    },
+    disabled: {
+      default: boolean('Disabled', false)
+    }
+  },
+  template: `
+  <Button
+    :variantColor="variant"
+    :size="size"
+    :isDisabled="disabled"
+    onClick={action('hello')
+  >
+    {{ text }}
+  </Button>`
```

Comment:
let's use `render` function instead 😄

Reply:
I really have no idea how to get that running with knobs 🙈

Reply:
the template code you wrote is actually broken

Reply:
Oh, yes - the missing curly bracket. That‘s what happens when the baby suddenly wakes up and you push stuff in a hurry 😅